dll/win32/shell32/shellmenu/CMenuBand.cpp

```diff
@@ -854,10 +855,8 @@ HRESULT CMenuBand::_TrackContextMenu(IContextMenu * contextMenu, INT x, INT y)
         _MenuItemSelect(MPOS_FULLCANCEL);
 
         TRACE("Before InvokeCommand\n");
-        CMINVOKECOMMANDINFO cmi = { 0 };
-        cmi.cbSize = sizeof(cmi);
-        cmi.lpVerb = MAKEINTRESOURCEA(uCommand);
-        cmi.hwnd = hwnd;
+        CMINVOKECOMMANDINFO cmi = { sizeof(cmi), 0, hwnd };
+        cmi.lpVerb = MAKEINTRESOURCEA(uCommand - idCmdFirst);
```

Comment:
Are you sure this is correct @katahiromz ?
Users of `IContextMenu` should very rarely modify id's like this...

Reply:
This value is an offset. It's not command ID in   general.